app/_src/policies/meshservice.md

```diff
@@ -0,0 +1,94 @@
+---
+title: MeshService
+---
+
+{% warning %}
+This resource is experimental!
+In Kubernetes, to take advantage of the automatic generation described below,
+you need to set both [control plane configuration variables](/docs/{{ page.version }}/reference/kuma-cp/) `KUMA_EXPERIMENTAL_SKIP_PERSISTED_VIPS`
+and `KUMA_EXPERIMENTAL_GENERATE_MESH_SERVICES` to `"true"` on the zone control
+planes that use `MeshServices`.
+{% endwarning %}
+
+MeshService is a new resource that represents what was previously expressed by
+the `Dataplane` tag `kuma.io/service`. Kubernetes users should think about it as
+the analog of a Kubernetes `Service`.
+
+A basic example follows to illustrate the structure:
+
+```
+type: MeshService
+name: redis
+mesh: default
+labels:
+  team: db-operators
+spec:
+  selector:
+    dataplaneTags: # tags in Dataplane object, see below
+      app: redis
+      k8s.kuma.io/namespace: redis-system # added automatically
+      kuma.io/zone: east-1 # added automatically
```

Comment:
nit: we diverged from this. `kuma.io/zone` is not added automatically.
Namespace is added by the controller so we can say it's added automatically